models.py

```diff
@@ -1136,6 +1136,12 @@ class FeatureForm(forms.Form):
                               'statuses In developer trial and Origin trial this '
                               'MUST be blank.')
 
+SUMMARY_PLACEHOLDER_TXT = (
+    'NOTE: This text describes this feature in the eventual beta release post '
+    'as well as possibly in other external places externally.&#10;&#10;'
```

Comment:
I don't think that &#10; works in this context.  Some of our existing code using \n to make a line break.  That works in Chrome, but not safari, which is probably still OK so long as the text is readable when it runs together.  A combination of \n and a space is probably better.

Reply:
When I was playing around with this in DevTools, the page wouldn't display \n, but it would display `&#10;.